content/java/concepts/array-list/terms/clear/clear.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: '.clear()'
+Description: 'Removes all elements from an ArrayList.' 
+Subjects:
+  - 'Java'
+  - 'Data Structures'
+Tags:
+  - 'Method'
+  - 'ArrayList'
+  - 'Java'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/java-developer'
+---
+
+The **`.clear()`** method removes all elements from an [`ArrayList`](https://www.codecademy.com/resources/docs/java/array-list) in Java. 
+
+## Syntax
+All elements of an instance of `ArrayList` can be cleared by passing the `.clear()` method. After invoking this method, the list will be empty, but the `ArrayList` instance remains with its current capacity unchanged. It does not return anything but rather empties the list.
```

Comment:
This particular line can be added in the introductory paragraph itself. You can remove it from here and add it in the above paragraph.